pkgs/tools/security/osv-scanner/default.nix

```diff
@@ -6,16 +6,20 @@
 }:
 buildGoModule rec {
   pname = "osv-scanner";
-  version = "1.2.0";
+  version = "1.3.0";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-5078mJbqiWu+Q0oOWaCJ8YUlSTRDLjmztAhtVyFlvN8=";
+    hash = "sha256-4iDuj+FM2EJKH/UspmCHqSypdTd3IXUdLQEsEI/SofY=";
   };
 
-  vendorHash = "sha256-LxwP1eK88H/XsGsu8YA3ksZnYJcOr7OzqWmZDRHO5kU=";
+  vendorHash = "sha256-MMEkgGyetwMEiD242CPYh619o4bo4zj87jnl7HvS0OE=";
+
+  subPackages = [
+    "cmd/osv-scanner"
```

Comment:
is there a reason to only build this package?

Reply:
There is now other packages in the repository that are not meant to be built (and cannot in our case because it is not the same vendor) and published.  They are used internally as test cases for the new Govulncheck integration.